k8s/aat/common/divorce/div-cms.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: div-cms
+  namespace: divorce
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: div-cms
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: div-cms
+    version: 1.1.0
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
+      image: hmctspublic.azurecr.io/div/cms:prod-f7412e3e
+      ingressHost: div-cms-aat.service.core-compute-aat.internal
+      environment:
+        AUTH_PROVIDER_SERVICE_CLIENT_BASEURL : "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+        DRAFT_STORE_API_BASEURL : "http://draft-store-service-aat.service.core-compute-aat.internal"
+        IDAM_CASEWORKER_USERNAME : "DivCaseWorker@AAT.com"
+        CASE_FORMATTER_SERVICE_API_BASEURL : "http://div-cfs-aat.service.core-compute-aat.internal"
+        IDAM_API_BASEURL: "https://idam-api.aat.platform.hmcts.net"
+        IDAM_API_REDIRECT_URL : "https://div-pfe-aat.service.core-compute-aat.internal/authenticated"
+        CASE_DATA_STORE_BASEURL : "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+        MANAGEMENT_ENDPOINT_HEALTH_CACHE_TIMETOLIVE: "30000"
+        DOCUMENTATION_SWAGGER_ENABLED: "true"
```

Comment:
Copied the above from (Apart from reform env and app insights) https://github.com/hmcts/div-case-maintenance-service/blob/master/charts/div-cms/values.aat.template.yaml

Reply:
this should be false for prod and overridden on aat

Reply:
Yeah, this is the AAT file so we're overriding it